.github/actions/bouf/action.yaml

```diff
@@ -0,0 +1,122 @@
+name: Run bouf
+description: Generates signed OBS install files and updater files
+inputs:
+  updaterPrivateKey:
+    description: Private key for signing update manifest
+    required: true
+  gcpWorkloadIdentityProvider:
+    description: GCP Identity Provider Pool ID
+    required: true
+  serviceAccountName:
+    description: Google service account name
+    required: true
+  version:
+    description: Version string (e.g. 30.0.0-rc1)
+    required: true
+  channel:
+    description: Update channel
+    required: false
+    default: 'stable'
+  commitHash:
+    description: Commit hash
+    required: false
+    default: ''
+
+runs:
+  using: composite
+  steps:
+    - name: Prepare Artifact
+      shell: pwsh
+      run: |
+        # Extract artifact in build dir and remove zip
+        cd build
+        Expand-Archive -Path *.zip -DestinationPath .
+        del *.zip
```

Comment:
I'd suggest to use `Remove-Item` - our PowerShell code mostly tries to keep using first-class PowerShell idioms as much as possible, only falling back on program invocations if necessary.